doc/gke-cluster-access.md

```diff
@@ -0,0 +1,74 @@
+# Setting up local access to GKE cluster
+
+In order to manage Production and Staging GKE clusters from your dev machine you will need to:
+- Authenticate with the `gcloud` CLI
+- Create a `kubectl` context for each cluster
+
+## Prerequisites
+
+The following tools need to be installed on your system:
+- [`kubectl`][install-kubectl] (this doc assumes **version >= 1.25**)
+- [`gcloud`][install-gcloud]
+- [`gke-gcloud-auth-plugin`][install-gke-auth-plugin] 
+
+[install-kubectl]: https://kubernetes.io/docs/tasks/tools/
+[install-gcloud]: https://cloud.google.com/sdk/docs/install
+[install-gke-auth-plugin]: https://cloud.google.com/blog/products/containers-kubernetes/kubectl-auth-changes-in-gke
+
+In your shell's environment, set
+
+```
+export USE_GKE_GCLOUD_AUTH_PLUGIN=True
+```
+
+You can do this either ad-hoc for your current session, or in your shell's `.rc` file.
+
+## Authenticating the `gcloud` CLI
+
+To authenticate the `gcloud` CLI, you need to run
+
+```
+gcloud auth login
+```
+
+which opens a browser that lets you log in using your target Google Account.
+Alternatively, pass `--no-browser` and follow the instructions in your terminal.
+
+If you have never used `gcloud` on your current setup before, you will also need to set the current project:
+
+```
+gcloud config set project wikibase-cloud
+```
+
+## Creating `kubectl` contexts
+
+After authenticating with `gcloud`, you can now create `kubectl` contexts for Staging:
+
+```
+gcloud container clusters get-credentials wbaas-2 --zone europe-west3-a
+```
+
+and Production
+
+```
+gcloud container clusters get-credentials wbaas-3 --zone europe-west3-a
+```
+
+## Using the contexts
+
+To use these contexts, use `kubectl config set`.
+E.g. in order to point your `kubectl` at staging, run:
+
+```
+kubectl config set-context gke_wikibase-cloud_europe-west3-a_wbaas-2
```

Comment:
```suggestion
kubectl config use-context gke_wikibase-cloud_europe-west3-a_wbaas-2
```

snipped from `kubectl config --help`:
```
get-contexts      Describe one or many contexts
  get-users         Display users defined in the kubeconfig
  rename-context    Rename a context from the kubeconfig file
  set               Set an individual value in a kubeconfig file
  set-cluster       Set a cluster entry in kubeconfig
  set-context       Set a context entry in kubeconfig
  set-credentials   Set a user entry in kubeconfig
  unset             Unset an individual value in a kubeconfig file
  use-context       Set the current-context in a kubeconfig file
```

set sounds like what you want at a glance but it it's seemingly more akin to "create context". `use` is the command for switching between contexts